runtime/gc_glue_java/CollectorLanguageInterfaceImpl.cpp

```diff
@@ -678,7 +678,8 @@ MM_CollectorLanguageInterfaceImpl::scavenger_switchConcurrentForThread(MM_Enviro
 			j9tty_printf(PORTLIB, "%p: Nursery [%p,%p] Evacuate [%p,%p] GS [%p,%p] Section size 0x%zx, sections %lu bit offset %lu bit mask 0x%zx\n",
 					vmThread, nurseryBase, nurseryTop, gsBase, gsTop, pageBase, pageTop, _extensions->getConcurrentScavengerPageSectionSize() , sectionCount, startOffsetInBits, bitMask);
 		}
-
+		vmThread->gsEvacuateBase = gsBase;
+		vmThread->gsEvacuateTop = gsTop;
```

Comment:
I'd remove 'gs' prefix from the two evacuate field names. They have nothing to do with GS facility.

Reply:
removed gs prefix